user-console/src/main/java/org/pentaho/mantle/client/admin/ChangePasswordDialog.java

```diff
@@ -152,11 +156,31 @@ public void onClick( ClickEvent event ) {
         cancelBtn.setEnabled( false );
         String newPassword = newPasswordTextBox.getText();
         String administratorPassword = administratorPasswordTextBox.getText();
+        String allowedChars = "a-z A-Z 0-9 _ . , : ; < > | ! @ # $ % ^ & * ( ) [ ]";
```

Comment:
This should be a static final and should be declared near the "regex variables" in line 45/46.